lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,65 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+      )
+
+      let :issue, object(
+        snippet: string?,
+      )
+    end
+
+    register_config_schema SCHEMA.config
+
+    DEFAULT_TARGET = "".freeze
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - myworkflow.yml
+          - workflows/anotherworflow.yml
+      YAML
+    end
+
+    def analyze(changes)
+      stdout, _stderr = capture3!(
+        analyzer_bin,
+        '-format', '{{json .}}',
+        '-shellcheck=',
+        '-pyflakes=',
+        '--',  # do not process any options after this
+        *Array(config_linter[:target] || DEFAULT_TARGET),
```

Comment:
[suggestion] Can we omit `DEFAULT_TARGET`? If arguments are omitted, `actionlint` seems to scan `.github/workflows/*.{yml,yaml}` files:

```suggestion
        *Array(config_linter[:target]),
```

Reply:
But does `config_linter[:target]` resolve to empty array if `target` is not specified?

Reply:
Yes.

```console
$ irb
irb(main):001:0> Array(nil)
=> []
irb(main):002:0> Array([])
=> []
```

Reply:
Ok, that makes sense. I removed the DEFAULT_TARGET.